### PR TITLE
Modulesync 1.2.0

### PR DESCRIPTION
**Pull request by TraGicCode**

### CODE REVIEW

File: .msync.yml [hunk 1]
```diff
@@ -1 +1 @@
-modulesync_config_version: '1.1.0'
+modulesync_config_version: '1.2.1'
```
Comment (wyardley) [inline .msync.yml]: The commit and the PR title should be 'modulesync [version]', and, as was pointed out to me the other day, you should check out the latest tag (1.2.0 in this case), and run the modulesync off of that.

Comment (TraGicCode) [reply .msync.yml]: Sure thing.  Any idea why we have to checkout a tag?

Comment (TraGicCode) [reply .msync.yml]: Made the appropriate changes.  I'm assuming i updated this a couple a weeks ago so this is probably a downgrade as far as msync goes.

Comment (wyardley) [reply .msync.yml]: Yeah, I thought the 1.2.0 tag got created, but it didn't.
I think basing it on the tag is just to ensure that the versioning is consistent (that is, if it says 1.2.0, it's 1.2.0 exactly and not 1.2.0++.

https://github.com/voxpupuli/modulesync_config#how-to-use-it

**Comment by bastelfreak** [comment]
A few things here look strange. Where does Coveralls::SimpleCov::Formatter come from? Is this from our current modulesync_config repo synced?

File: Rakefile [hunk 2]
```diff
@@ -31,17 +31,6 @@ task test: [
   :release_checks,
 ]
 
-desc "Run main 'test' task and report merged results to coveralls"
```
Comment (alexjfisher) [inline Rakefile]: This looks like you're undoing the latest modulesync changes I made in https://github.com/voxpupuli/modulesync_config/commit/8393b3267da937c83814ab24883ceba2fae5ce0e#diff-34d8b3ef11f42c86a299a165682095bd

**Comment by alexjfisher** [comment]
PR title has been changed to 'modulesync 1.2.0', but commit is for 'modulesync 1.1.0'

**Comment by TraGicCode** [comment]
Sorry guys.  I'll be on it in the morning

**Comment by TraGicCode** [comment]
@alexjfisher Done.

**Pull request merged**